cachito/web/content_manifest.py

```diff
@@ -0,0 +1,84 @@
+# SPDX-License-Identifier: GPL-3.0-or-later
+import flask
+
+
+class ContentManifest:
+    """A content manifest associated with a Cacihto request."""
+
+    version = 1
+    json_schema_url = (
+        "https://raw.githubusercontent.com/containerbuildsystem/atomic-reactor/"
+        "f4abcfdaf8247a6b074f94fa84f3846f82d781c6/atomic_reactor/schemas/content_manifest.json"
+    )
+    unknown_layer_index = -1
+
+    def __init__(self, request=None):
+        """
+        Initialize ContentManifest.
+
+        :param Request request: the request to generate a ContentManifest for
+        """
+        self.request = request
+        # dict to store go package level data; uses the package purl as key to identify a package
+        self._gopkg_data = {}
+
+    def process_go_package(self, package, dependency):
+        """
+        Process gomod or go-package package.
+
+        :param Package package: the gomod or go-package package to process
+        :param Dependency dependency: the gomod or go-package package dependency to process
+        """
+        purl = package.to_purl()
+        # self._gopkg_data.setdefault(purl, {"purl": purl, "dependencies": [], "sources": []})
```

Comment:
Could you please remove this unnecessary comment?

Reply:
done